include/deal.II/grid/tria.h

```diff
@@ -2743,6 +2744,13 @@ class Triangulation : public Subscriptor
   active_cell_iterator
   last_active() const;
 
+  /**
+   * Return an iterator to a cell of this Triangulation object constructed from
+   * an independent CellId object.
```

Comment:
Maybe add a sentence like this:
```suggestion
   * an independent CellId object. If the given argument corresponds to a valid
   * cell in this triangulation, this operation will always succeed for sequential
   * triangulations where the current processor stores all cells that are part
   * of the triangulation. On the other hand, if this is a parallel triangulation,
   * then the current processor may not actually know about this cell and in this
   * case ...
```
+ add what actually happens. I suspect that the operation is ok if it's a locally owned or ghost cell, but may not work if it's an artificial cell. Maybe say so.

Reply:
I suspect so, too. Artificial cells may not be refined enough to create a cell iterator. I've adjusted the documentation and the error message accordingly.